### DISCUSSION TITLE
Im getting cors eror using react and vite.js

### DISCUSSION THREAD

**Discussion by phi2505** [supabase/supabase#16111] [Questions]
Have anybody had the same issue? I just suddelnly started having cors eror on using the supabase functions, like:  const { data: { user } } = await supabase.auth.getUser() 

![Skjermbilde 2023-07-29 165308](https://github.com/supabase/supabase/assets/104219507/10f6f300-8591-4d0e-9afb-6a5c08921d92)
![Skjermbilde 2023-07-29 165423](https://github.com/supabase/supabase/assets/104219507/661ce899-233a-4efe-be9b-5c16328e73fe)

**Comment by GaryAustin1** [comment]
You might look at the jwt length based on this report:  https://github.com/supabase/supabase/issues/7627
